pkg/multiboot/internal/trampoline/trampoline_linux_amd64.s

```diff
@@ -0,0 +1,154 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// We want all the trampoline's assembly code to be located
+// in a contiguous byte range in a compiled binary.
+// Go compiler does not guarantee that. Still, current version
+// of compiler puts all pieces together.
+		
+#include "textflag.h"
+
+#define MSR_EFER	0xC0000080
+#define EFER_LME	0xFFFFFEFF
+#define CR0_PG		0x0FFFFFFF
+
+#define	DATA_SEGMENT	0x00CF92000000FFFF
```

Comment:
The alignment is a bit off here.

Reply:
Strange, in my github looks fine, removed tab